plugins/utils/autosource.nix

```
@@ -0,0 +1,24 @@
+{
+  helpers,
```

`helpers` is deprecated. We can use `lib.nixvim` instead.

```suggestion
  lib,
```